libpromises/locks.c

```diff
@@ -757,7 +743,7 @@ CfLock AcquireLock(EvalContext *ctx, const char *operand, const char *host,
         if (elapsedtime < tc.ifelapsed)
         {
             Log(LOG_LEVEL_VERBOSE,
-                "XX Nothing promised here [%.40s] (%jd/%u minutes elapsed)",
+                "Nothing promised here [%.40s] (%jd/%u minutes elapsed)",
```

Comment:
This is ok, but I would refrain from making changes like this, so far away from what you're actually working on. Especially as part of the commit where you are making the real changes.